tests/autoprint.R

```diff
@@ -1,4 +1,4 @@
-require(data.table)
+suppressPackageStartupMessages(require(data.table))
```

Comment:
For this PR I'd prefer `require(data.table, exclude="%notin%")`, however that's from 3.6.0 (r76248):

https://github.com/r-devel/r-svn/commit/c220e47d6012f220f6e7b7345ded4d7facfb9282

Maybe just `require(data.table, warn.conflicts=!exists("%notin%", "package:base"))` is the most concise way to narrowly skip for only this issue?

With blanket `suppress*` I worry about missing other such issues in the future.

Reply:
That's very concise compared to dynamic call manipulation and testing for `'exclude' %in% names(formals(require)))`, thank you!

Reply:
yea, still not ideal, as it means all future versions turn off conflict warnings. I guess it doesn't matter much for this test anyway -- we could just statically set it to FALSE?